orc8r/gateway/python/magma/magmad/metrics_collector.py

```diff
@@ -49,19 +65,26 @@ def __init__(self, services: List[str],
         self._loop = loop if loop else asyncio.get_event_loop()
         self._retry_queue = []
         self._samples = []
-        self._grpc_options = _get_metrics_chan_grpc_options(grpc_max_msg_size_mb)
+        self._grpc_options = _get_metrics_chan_grpc_options(
+            grpc_max_msg_size_mb)
```

Comment:
https://github.com/magma/magma/wiki/Contributing:-Python-Code-Conventions#line-length and so forth for other changes